src/box/errcode.h

```diff
@@ -115,286 +115,286 @@ struct errcode_record {
+	_(ER_DROP_USER, 44,			"Failed to drop user", "user", STRING, "details", STRING) \
+	_(ER_NO_SUCH_USER, 45,			"User does not exist", "user", STRING) \
+	_(ER_USER_EXISTS, 46,                   "User already exists", "user", STRING) \
 	_(ER_CREDS_MISMATCH, 47,		"User not found or supplied credentials are invalid") \
-	_(ER_UNKNOWN_REQUEST_TYPE, 48,		"Unknown request type %u") \
-	_(ER_UNKNOWN_SCHEMA_OBJECT, 49,		"Unknown object type '%s'") \
-	_(ER_CREATE_FUNCTION, 50,		"Failed to create function '%s': %s") \
-	_(ER_NO_SUCH_FUNCTION, 51,		"Function '%s' does not exist") \
-	_(ER_FUNCTION_EXISTS, 52,		"Function '%s' already exists") \
+	_(ER_UNKNOWN_REQUEST_TYPE, 48,		"Unknown request type", "request_type", UINT) \
+	_(ER_UNKNOWN_SCHEMA_OBJECT, 49,		"Unknown object type", "object_type", STRING) \
+	_(ER_CREATE_FUNCTION, 50,		"Failed to create function", "function", STRING, "details", STRING) \
+	_(ER_NO_SUCH_FUNCTION, 51,		"Function does not exist", "function", STRING) \
+	_(ER_FUNCTION_EXISTS, 52,		"Function already exists", "function", STRING) \
 	_(ER_BEFORE_REPLACE_RET, 53,		"Invalid return value of space:before_replace trigger: expected tuple or nil") \
-	_(ER_MULTISTATEMENT_TRANSACTION, 54,	"Can not perform %s in a multi-statement transaction") \
-	_(ER_TRIGGER_EXISTS, 55,		"Trigger '%s' already exists") \
-	_(ER_USER_MAX, 56,			"A limit on the total number of users has been reached: %u") \
-	_(ER_NO_SUCH_ENGINE, 57,		"Space engine '%s' does not exist") \
-	_(ER_RELOAD_CFG, 58,			"Can't set option '%s' dynamically") \
-	_(ER_CFG, 59,				"Incorrect value for option '%s': %s") \
+	_(ER_MULTISTATEMENT_TRANSACTION, 54,	"Can not perform in a multi-statement transaction", "action", STRING) \
+	_(ER_TRIGGER_EXISTS, 55,		"Trigger already exists", "trigger", STRING) \
+	_(ER_USER_MAX, 56,			"A limit on the total number of users has been reached", "limit", UINT) \
+	_(ER_NO_SUCH_ENGINE, 57,		"Space engine does not exist", "engine", STRING) \
+	_(ER_RELOAD_CFG, 58,			"Can't set option dynamically", "option", STRING) \
+	_(ER_CFG, 59,				"Incorrect value for option", "option", STRING, "details", STRING) \
 	_(ER_SAVEPOINT_EMPTY_TX, 60,		"Can not set a savepoint in an empty transaction") \
-	_(ER_NO_SUCH_SAVEPOINT, 61,		"Can not rollback to savepoint: the savepoint does not exist") \
-	_(ER_UNKNOWN_REPLICA, 62,		"Replica %s is not registered with replica set %s") \
-	_(ER_REPLICASET_UUID_MISMATCH, 63,	"Replica set UUID mismatch: expected %s, got %s") \
-	_(ER_INVALID_UUID, 64,			"Invalid UUID: %s") \
+	_(ER_NO_SUCH_SAVEPOINT, 61,		"Savepoint does not exist") \
+	_(ER_UNKNOWN_REPLICA, 62,		"Replica is not registered with replica set", "replica", STRING, "replicaset", STRING) \
+	_(ER_REPLICASET_UUID_MISMATCH, 63,	"Replica set UUID mismatch", "expected", STRING, "actual", STRING) \
+	_(ER_INVALID_UUID, 64,			"Invalid UUID", "uuid", STRING) \
 	_(ER_REPLICASET_UUID_IS_RO, 65,		"Can't reset replica set UUID: it is already assigned") \
```

Comment:
"Cannot reset replica set UUID because it is already assigned"

Reply:
Fixed. 

```diff
+       _(ER_REPLICASET_UUID_IS_RO, 65,         "Cannot reset replica set UUID because it is already assigned") \
```